packages/workspace/src/utilities/prettier.ts

```diff
@@ -1,31 +1,32 @@
+import { resolveConfig, resolveConfigFile } from 'prettier';
 import type { Options } from 'prettier';
-import * as cosmiconfig from 'cosmiconfig';
+import { NormalModuleReplacementPlugin } from 'webpack';
 
 export interface ExistingPrettierConfig {
   sourceFilepath: string;
   config: Options;
 }
 
-export function resolveUserExistingPrettierConfig(): Promise<ExistingPrettierConfig | null> {
-  const explorer = cosmiconfig('prettier', {
-    sync: true,
-    cache: false,
-    rcExtensions: true,
-    stopDir: process.cwd(),
-    transform: (result) => {
-      if (result && result.config) {
-        delete result.config.$schema;
-      }
-      return result;
-    },
-  });
-  return Promise.resolve(explorer.load(process.cwd())).then((result) => {
-    if (!result) {
+export async function resolveUserExistingPrettierConfig(): Promise<ExistingPrettierConfig | null> {
+  try {
+    const filepath = await resolveConfigFile();
+    if (!filepath) {
       return null;
     }
+
+    const config = await resolveConfig(process.cwd(), {
+      useCache: false,
+      config: filepath,
+    });
+    if (!config) {
+      return null;
+    }
+
     return {
-      sourceFilepath: result.filepath,
-      config: result.config,
+      sourceFilepath: filepath,
+      config: config,
     };
-  });
+  } catch {
+    return null;
+  }
```

Comment:
@meeroslav I did a bit of research and the problems may have come from the `cosmiconfig` module. I then read the prettier docs and found the 2 methods [`resolveConfigFile()`](https://prettier.io/docs/en/api.html#prettierresolveconfigfilefilepath) and [`resolveConfig()`](https://prettier.io/docs/en/api.html#prettierresolveconfigfilepath--options) which are the right choice for the method instead of making a custom cosmiconfig call.

Reply:
These changes should fix all tests at least they fixed all tests locally...